examples/fluent-bit/cloudwatchlogs/README.md

```diff
@@ -31,3 +31,17 @@ This field instructs the CloudWatch plugin to only send the value of the log key
 		}
 	},
 ```
+
+			"log_key": "log",
+The following table shows how your logs will appear in CloudWatch with and without `log_key` and `enable-ecs-log-metadata` if "`my_sample log`" is printed to stdout:
+
+|Add logConfiguration.options|Add firelensConfiguration.options|received by cloudwatch|description|comments|
```

Comment:
Why does each title have "Add" in it, why not just "logConfiguration.options" or spell it out as more of a complete thought like "Add to logConfiguration.options" or "logConfiguration.options contains"